backend/Tasque.Core.WebAPI/appsettings.json

```diff
@@ -1,38 +1,32 @@
 {
-  "Host": {
-    "Logo": "http://16.170.225.154/assets/tasque_logo.svg",
-    "DarkLogo": "http://16.170.225.154/assets/tasque_logo_dark.svg",
-    "BigLogo": "http://16.170.225.154/assets/tasque_big_logo.png"
-  },
   "Logging": {
     "LogLevel": {
       "Default": "Information",
       "Microsoft.AspNetCore": "Warning"
     }
   },
   "JwtIssuerOptions": {
-    "Issuer": "",
-    "Audience": "",
+    "Issuer": "shcho-zavhodno",
+    "Audience": "holovne-shchob-ne-pusto",
     "ValidFor": "1",
     "Key": "BSA-2022-Tasque.Core"
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "TasqueDb": "Host=localhost; Database=tasquedb; Username=postgres; Password=mysecretpassword"
+    "TasqueDb": "Host=localhost; Database=tasquedb; Username=postgres; Password=123"
   },
   "MailJetOptions": {
     "ApiKey": "",
     "ApiSecret": "",
     "IsSandboxMode": false
   },
   "SendGridOptions": {
-    "ApiKey": ""
+    "ApiKey": "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"
```

Comment:
It seems not worth committing the key to the repository. It is better to use a secret store